js-foundations.js

```diff
@@ -4,22 +4,61 @@
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
 
 
+var divisible = (number) => {
+    if(number % 3){
+        return "is divisible by three"
+    }else{ 
+         return "is not divisible by three."
+    }
+}
+
+console.log(divisible(15,0,385-7))
+
+
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
 
+var  helloMe = {
+    name:"Sabrina",
+    eyes: "blue", 
+    hair: "blonde"
+}
 
+console.log(helloMe.name)
+console.log(helloMe.eyes)
+console.log(helloMe.hair)
+
+console.log(`${helloMe.name} has ${helloMe.eyes} eyes and ${helloMe.hair} hair.`)
 
 // ------------------------------------- Consider the variable:
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 
 // 3a. Write a function that returns every other item from the array as one string. Expected output: "streetlamp teeth nephew database"
 
+function drop(array){
+    let newArr=[]
+    for(let i=0; i<array.length; i++){
+        if (array[i]=== "streetlamp","teeth","nephew","database"){
+            newArr.push(array[i])
+        }
+    }
+    return newArr
+}
+console.log(drop(randomNouns))
 
```

Comment:
I'm getting the full array back from this code. You have the right idea, but we need a evaluation for every other index.